src/shared/containers/admin/advanced.jsx

```diff
@@ -0,0 +1,286 @@
+/**
+ * Copyright (c) 2015-present, CWB SAS
+ *
+ * This source code is licensed under the GPL v2.0+ license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import Zrmc, { Grid, Inner, Cell, Button, Icon, TextField } from "zrmc";
+import { connect } from "react-redux";
+import {
+  apiSetAdminParametersRequest,
+  apiAdminUpdateRequest,
+} from "../../actions/api";
+import TunnelBox from "../../components/tunnelBox";
+import { infoStyleD, FORM_WIDTH } from "./styles";
+
+class Advanced extends Component {
+  constructor(props) {
+    super(props);
+
+    const { params } = props.admin;
+    this.state = {
+      tunnelParams: null,
+      backendParams: {
+        publicUrl: "",
+        apiUrl: params.backend.apiUrl || "",
```

Comment:
Don't see why but I have this apiUrl / authUrl:
![screen shot 2018-05-24 at 16 09 51](https://user-images.githubusercontent.com/395718/40490902-110dbdfe-5f6d-11e8-979f-c1931b91288b.png)

Reply:
this is a backend issue, not zoapp nor opla front